protostar/commands/test/test_runner.py

```diff
@@ -130,7 +130,7 @@ async def _run_test_suite(
 
             if test_suite.setup_fn_name:
                 await invoke_setup(
-                    test_suite.setup_fn_name, execution_state, setup_stdout_buffer
+                    test_suite.setup_fn_name, execution_state, setup_stdout
```

Comment:
You shouldn't pass output name from here down to `perform_invoke`. `ExecutionEnvironment` should by itself decide what name this output will have. In other words, please, remove the `output_name` parameter and simply hardcode `"setup"` in `SetupExecutionEnvironment` etc.